### DISCUSSION TITLE
Discord server for contribution management

### DISCUSSION THREAD

**Discussion by Gargamiel** [Gargamiel/PokeWorld#21] [Announcements]
@Nolenz @lmgginspace If you want to contribute to the mod there's a new Discord server setup at https://discord.gg/Y6972kAdup with a channel dedicated to contributors, this way we can better manage collaboration.

**Comment by Gargamiel** [comment]
@Nolenz I was wondering what tool you used to create a nice outline around all the Pokémon textures. I have a new set of textures on which I would like to apply the same outline. If you have the time I could send them to you, but if you don't it would be great if you would tell me your secret. Thanks.

**Reply by Nolenz** [reply]
Just a layer style in Photoshop and a macro to apply it to every image and save. I'd be happy to apply it to any new sprites you'd like.

![Layer Style Settings](https://i.imgur.com/Nkjk6I7.png)

**Comment by Palisar** [comment]
> @Nolenz @lmgginspace If you want to contribute to the mod there's a new Discord server setup at https://discord.gg/xSh5bA3C with a channel dedicated to contributors, this way we can better manage collaboration.

This link had expired. I would like to join and try and contribute!

**Reply by Gargamiel** [reply]
There you have it, this one shouldn't expire https://discord.gg/Y6972kAdup
